### DISCUSSION TITLE
Ticwatch Pro 5 & xDrip

### DISCUSSION THREAD

**Discussion by bjblackmore** [NightscoutFoundation/xDrip#3511] [Q&A] [answered]
Hi,
I just upgraded from my original Ticwatch Pro, which I've had for 4 or 5 years,  to the new Ticwatch Pro 5. However I'm having difficulty getting Xdrip installed. The old method doesn't seem to work as Ticwatch no longer uses Wear OS and insists you uninstall it, and use Mobvoi Health app instead.
Has anyone successfully managed to install Xdrip on a new Ticwatch/Ticwatch 5?

**Comment by bjblackmore** [comment, marked as answer]
Got it working! Had to enable debugging  over wireless and pair the new device for Wear Installer 2 to install the app.

**Reply by yuvalutilis** [reply]
Amazing.... worked for me too... been struggling with this for a few months. Yhanks @bjblackmore

**Reply by tshakah** [reply]
Does it work in standalone mode?
